reactnative/ReactNativeTemplateApp/src/com/salesforce/samples/reactnativetemplateapp/ReactNativeTemplateApp.java

```diff
@@ -28,14 +28,41 @@
 
 import android.app.Application;
 
+import com.facebook.react.ReactApplication;
+import com.facebook.react.ReactNativeHost;
+import com.facebook.react.ReactPackage;
+import com.facebook.react.shell.MainReactPackage;
 import com.salesforce.androidsdk.app.SalesforceSDKManager;
 import com.salesforce.androidsdk.reactnative.app.SalesforceReactSDKManager;
 import com.salesforce.androidsdk.security.Encryptor;
 
+import java.util.Arrays;
+import java.util.List;
+
 /**
  * Application class for our application.
  */
-public class ReactNativeTemplateApp extends Application {
+public class ReactNativeTemplateApp extends Application implements ReactApplication {
+
+	private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
+		@Override
+		protected boolean getUseDeveloperSupport() {
+			return BuildConfig.DEBUG;
```

Comment:
I see this overridden in the `Activity` class too [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/1212/files#diff-83034e6e1c67604065fef6a2ec76c8afR58). Is that still required?